src/main/java/org/jabref/model/entry/types/BibtexEntryTypeDefinitions.java

```diff
@@ -20,12 +20,12 @@ public class BibtexEntryTypeDefinitions {
      * An article from a journal or magazine.
      * <p>
      * Required fields: author, title, journal, year.
-     * Optional fields: volume, number, pages, month, note.
+     * Optional fields: volume, number, pages, month, note, langid.
```

Comment:
This is a biblatex field, not a BibTeX field. Please revert. --> `git checkout main -- src/main/java/org/jabref/model/entry/types/BibtexEntryTypeDefinitions.java`

Reply:
Yes my apologies, I was working in the BibTex class before noting the difference between the two